README.md

```diff
@@ -278,13 +296,13 @@ The production components are:
     (command-line and RPC) for directly interacting with the Chainweb or for
     implementing applications such as miners and transaction management tools.
 
-*   `chainweb-miner`: A stand-alone Mining Client.
-
 *   `chainweb-tests`: A test suite for the Chainweb library and chainweb-node.
 
-### Architecture Overview
+*   `cwtool`: A collection of random tools that are helpfule for maintaining, testing,
+    and debugging chainweb.
 
-For a detailed description of the Kadena architecture,
-[see here](docs/Architecture.md).
+*   `bench`: a collection of benchmarks
+
+### Architecture Overview
 
 ![Architecture Overview](docs/Overview.png)
```

Comment:
This diagram is outdated. Do we have a high quality version of the new component diagram from the "Kadena Teaches" videos?